phpphp/homework3.php

```diff
@@ -0,0 +1,19 @@
+<?php
+//homework3
+function checkPrimeNumber ($n) {
+  if ($n === 0 || $n === 1)  {
+    return false;
+  }
+  if ($n === 2) {
+    return true;
+  } 
+  $m = $n/2;
+  for($i = 2; $i <= $m; ++$i) {
+      if ($n%$i == 0) {
+        return false;
+        break;  
+      }
+  }
+  return true;
+}
```

Comment:
indent sometimes 2 spaces, sometimes 1 tab (4 spaces)
can you fix only 1 format?